pallets/task/src/lib.rs

```diff
@@ -103,7 +104,7 @@ pub mod pallet {
 		pub current_owner: AccountOf<T>,
 		pub status: TaskStatus,
 		pub budget: BalanceOf<T>,
-		pub deadline: u32,
+		pub deadline: u64,
```

Comment:
why change to u64?

Reply:
Duration::from_millis() takes `u64` as input.